Bio/SwissProt/__init__.py

```diff
@@ -16,6 +16,18 @@
 
 """
 
+# flake8: noqa
```

Comment:
Why is this being used? I think it turns off flake8 for the entire file which is a very bad idea.